tests/qm-sanity-test/README.md

```diff
@@ -0,0 +1,17 @@
+# QM sanity test
+
+This test suite do some basic sanity tests for qm to confirom that qm has been installed properly and the qm environment has been started successfully.
+
+## This Test Suite includes these tests
+
+1. Confirm that qm is up and running.
+
+2. Confirm that podman in qm is ok.
+
+3. Confirm that bluechi-controller is active and bluechi-agent is online.
+
+4. Confirm that ASIL and QM bluechi-agent are connected to bluechi-controller.
```

Comment:
Please change all ASIL -> host 
We do not want to mislead people that this setup is FuSA certified

Reply:
done